app/src/main/java/org/oppia/app/player/state/CustomCheckbox.kt

```diff
@@ -0,0 +1,69 @@
+package org.oppia.app.player.state
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.text.Spanned
+import android.view.View
+import android.widget.CheckBox
+import android.widget.CompoundButton
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.content.ContextCompat
+import org.oppia.app.R
+import org.oppia.util.data.UrlImageParser
+import android.content.res.ColorStateList
+import android.os.Build
+import android.view.Gravity
+
+// TODO(#163): Move this to a StateCardFragment Low-fi PR.
+/** Custom Checkbox for MultipleSelectionInputInteractionView. */
+class CustomCheckbox(context: Context, private val optionContents: String) : CheckBox(context) {
```

Comment:
Instead of ``CustomCheckbox`` and ``CustomRadioButton`` maybe instead: ``HtmlCheckbox`` and ``HtmlRadioButton`` might be a bit clearer as to what these views are doing & how they should be used.